validation-authority.go

```diff
@@ -15,8 +15,12 @@ import (
 	"io/ioutil"
 	"net/http"
 	"time"
+	"log"
 )
 
+// Set this to true when testing locally, false otherwise.
+const UseLocalhost = true
```

Comment:
I would generally like to avoid compile-time flags like this, and instead use an environment variable or a command line parameter.